Tutorial Proposal: Python toolchain for deploying a release build

cwinge opened the pull request:
# Tutorial Proposal - Python toolchain for deploying a release build

## Members
* Balthazar West, bwest@kth.se, Github: balwes

* Christer Winge, cwing@kth.se, Github: cwinge

## Tutorial topic
We propose to create a tutorial that showcase a possible toolchain of helpful tools that would assist in making sure the application is in a desired state at the end of development.

This would include tools that handle the following key points
* code analysis
* testing
* obfuscation (and minimization)
* compilation

### Goal
With a completed tutorial, one should be able to chain these tools together, making it a very effective process that is essentially automated.

cesarsotovalero commented:
@cwinge: your proposal looks ambitions. It would be better to focus in-depth on a single key point (e.g. obfuscation). If you plan to go ahead with this tutorial, please be more explicit in describing the tool that you'll use in each point of the pipeline.

cwinge commented:
@cesarsotovalero Thank you for the feedback. I have updated the README to reflect on the changes we have done for the proposal, I'll paste it below as well. In short: we have taken your advice and focused on one key point, being obfuscation, where we will showcase and compare some tools used for Python.

I have also added a README for the competition, but I can't create a separate pull request since this is open. I hope you can help me with that anyway. I know the deadline is today, I'm just registering late.

------
# Tutorial Proposal - Code obfuscation tools for Python

## Members
* Balthazar West, bwest@kth.se, Github: balwes

* Christer Winge, cwing@kth.se, Github: cwinge

## Tutorial topic
We propose to create a tutorial that covers different code obfuscation tools available for Python, their techniques and differences as well as showcase some results obtained from using them.

The current tools that will be examined and covered(might change with more research done on the subject)
* pyarmor
* cython
* py-minifier

### Goal
After going through the tutorial, the reader should have a better understand about what tools are available for Python obfuscation, why it might be desirable to obfuscate code, as well as a good idea of how obfuscated code might look.

cesarsotovalero commented:
@cwinge: Your tutorial proposal looks good now. 
Also welcome to the fuzzing competition.
I'm merging this PR :+1:
